### ISSUE TITLE
[QuizQuestion] Render the label of answers as a div

### ISSUE THREAD

**Issue by huyenltnguyen** [freeCodeCamp/ui#330]
## Description

The label of the answers should be rendered as a `div` instead of a `p`, otherwise there would be an a11y issue when the label contains `PrismFormatted` (which could render as a `div`).

<details>
  <summary>Error when the label contains PrismFormatted</summary>

<img width="566" alt="Screenshot 2024-09-23 at 16 15 25" src="https://github.com/user-attachments/assets/091c63af-2583-4cdc-b66d-ff4b04cdfa72">
</details>

## Changes needed

Remove `as="p"` from this line:

https://github.com/freeCodeCamp/ui/blob/6cc3039debdfdbfe0c88f2aab8187af6b23436c5/src/quiz-question/answer.tsx#L92
